web.go

```diff
@@ -131,8 +132,19 @@ func (provider *WebProvider) Provide(configurationChan chan<- types.ConfigMessag
 	return nil
 }
 
+// healthResponse combines data returned by thoas/stats with statistics (if
+// they are enabled).
+type healthResponse struct {
+	*stats.Data
+	Statistics interface{} `json:"statistics,omitempty"`
+}
```

Comment:
What do you think of doing this instead:
```
type healthResponse struct {
	*stats.Data
	statistics
}

type statistics interface{}
```

This would allow API to give:
```
{
  // Træfɪk PID
  "pid": 2458,
  // Træfɪk server uptime (formated time)
  "uptime": "39m6.885931127s",
  //  Træfɪk server uptime in seconds
  "uptime_sec": 2346.885931127,
  // current server date
  "time": "2015-10-07 18:32:24.362238909 +0200 CEST",
  // current server date in seconds
  "unixtime": 1444235544,
  // count HTTP response status code in realtime
  "status_code_count": {
    "502": 1
  },
  // count HTTP response status code since Træfɪk started
  "total_status_code_count": {
    "200": 7,
    "404": 21,
    "502": 13
  },
  // count HTTP response
  "count": 1,
  // count HTTP response
  "total_count": 41,
  // sum of all response time (formated time)
  "total_response_time": "35.456865605s",
  // sum of all response time in seconds
  "total_response_time_sec": 35.456865605,
  // average response time (formated time)
  "average_response_time": "864.8016ms",
  // average response time in seconds
  "average_response_time_sec": 0.8648016000000001,
  // ten most recent requests with 4xx and 5xx status codes
  "recent_errors": [
      {
        // status code
        "status_code": 500,
        // description of status code
        "status": "Internal Server Error",
        // request HTTP method
        "method": "GET",
        // request hostname
        "host": "localhost",
        // request path
        "path": "/path",
        // RFC 3339 formatted date/time
        "time": "2016-10-21T16:59:15.418495872-07:00"
      }
  ]
}
```

This would allow us to transparently migrate from https://github.com/thoas/stats to our own stats middleware.